app/models/session.rb

```diff
@@ -3,6 +3,7 @@ class Session < ApplicationRecord
 
   validates :ip, presence: true
 
+  after_commit { GeoIPBuilderJob.new(String(ip)).enqueue }
```

Comment:
@ksylvest Out of curiosity, what happens if you don't type-cast `ip` (of inet class) into a String?
Active Job is not able to serialize and deserialize correctly? Is this a gap in Active Job / Sidekiq?

Reply:
ActiveJob isn't able to serialize with sidekiq (probably work without sidekiq).